event_routing_backends/processors/xapi/event_transformers/enrollment_events.py

```diff
@@ -14,9 +14,24 @@ class BaseEnrollmentTransformer(XApiTransformer):
     """
     Base transformer for enrollment events.
     """
-    additional_fields = ()
+    additional_fields = ('context', )
     event_version = 1.0
 
+    def get_context(self):
```

Comment:
@rehanedly I think we should define `get_context` method in `XApiTransformer` something like this
```suggestion
   def get_context(self):
        """
        Get context for xAPI transformed event.

        Returns:
            `Context`
        """

        context = Context(
            contextActivities=self.get_context_activities()
            extensions=self.get_context_extensions()
        )
        return context
```
and `get_context_extensions` method in `XApiTransformer` something like this
```suggestion
    def get_context_extensions(self):
        return Extensions({
                constants.XAPI_EVENT_VERSION_KEY: self.event_version
            })
```
That way we don't need to define `get_context` inside each transformer. If any child transformer need to change extensions or activities they can override relevant methods.